tests/nat/endpointIndependentNAT.test.ts

```diff
@@ -1,19 +1,21 @@
 import os from 'os';
 import path from 'path';
 import fs from 'fs';
-import process from 'process';
-import shell from 'shelljs';
 import Logger, { LogLevel, StreamHandler } from '@matrixai/logger';
 import * as testNatUtils from './utils';
-import { runDescribeIf } from '../utils';
+import { testIf } from '../utils';
```

Comment:
This should be `import * as testsUtils from '../utils';`. Should be consistent with the rest of usage of utils.